components/usage/pkg/db/workspace.go

```diff
@@ -51,6 +51,14 @@ func (d *Workspace) TableName() string {
 	return "d_b_workspace"
 }
 
+type WorkspaceType string
+
+const (
+	WorkspaceType_Prebuild WorkspaceType = "prebuild"
+	WorkspaceType_Probe    WorkspaceType = "probe"
```

Comment:
FYI, I believe the type `probe` is deprecated. Although, maybe there are still old occurrences in our DB, so it's probably fine to keep it here as well.